lib/MetaCPAN/Web/Test/HTML5/TreeBuilder.pm

```diff
@@ -0,0 +1,85 @@
+package MetaCPAN::Web::Test::HTML5::TreeBuilder;
+use strict;
+use warnings;
+
+use parent 'HTML::TreeBuilder';
+
+use MetaCPAN::Web::Test::HTML5::Element::SVG;
+use constant SVG_CLASS => 'MetaCPAN::Web::Test::HTML5::Element::SVG';
+
+my @svg_elements = qw(
+    a altGlyph altGlyphDef altGlyphItem animate animateColor animateMotion
+    animateTransform audio canvas circle clipPath color-profile cursor defs desc
+    discard ellipse feBlend feColorMatrix feComponentTransfer feComposite
+    feConvolveMatrix feDiffuseLighting feDisplacementMap feDistantLight
+    feDropShadow feFlood feFuncA feFuncB feFuncG feFuncR feGaussianBlur feImage
+    feMerge feMergeNode feMorphology feOffset fePointLight feSpecularLighting
+    feSpotLight feTile feTurbulence filter font-face-format font-face-name
+    font-face-src font-face-uri font-face font foreignObject g glyph glyphRef
+    hatch hatchpath hkern iframe image line linearGradient marker mask mesh
+    meshgradient meshpatch meshrow metadata missing-glyph mpath path pattern
+    polygon polyline radialGradient rect script set solidcolor stop style svg
+    switch symbol text textPath title tref tspan unknown use video view vkern
+);
```

Comment:
It might be nice to document in a comment where these element names came from, so they can be kept up to date if necessary.